tasks.py

```diff
@@ -186,13 +186,14 @@ def validate_kind_version():
     "protocol": "Pre-configure MetalLB with the specified protocol. "
                 "Unconfigured by default. Supported: 'bgp','layer2'",
```

Comment:
Can you add a description of this option? I know it is in the general section below but I think convention would also put it in the `@task(help={` section

Reply:
I missed that. Will do!

Reply:
Thanks, an example image name would be useful as I am still trying to find an example to test.

Reply:
You can find them here https://github.com/kubernetes-sigs/kind/releases
I tried and it seemed to work :P

Reply:
you could add something like `(e.g. kindest/node:v1.18.19)`

Reply:
yes will do

Reply:
Thanks, just tested it there and seems to work! cool.

Reply:
Done